android/build.gradle

```
@@ -23,6 +23,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'org.jetbrains.kotlin.android'
 
 android {
+    namespace 'com.mapbox.mapboxgl'
```

To stay compatible with older AGP we should make this conditional for now.

```suggestion
    if (project.android.hasProperty("namespace")) {
        namespace 'com.mapbox.mapboxgl'
    }
```